js_is-json.js

```diff
@@ -0,0 +1,13 @@
+function isJSON(str) {
+  try {
+      JSON.parse(str);
+      return true;
+  } catch (e) {
+      return false;
+  }
```

Comment:
краще повертати тут ніж на 4 рядку
```suggestion
  }
   return true;
```